benchmark/Benchmark.h

```diff
@@ -100,14 +157,24 @@ class BenchmarkRecords {
      *  Most of the time a lambda, that calls the actual function to benchmark
      *  with the needed parameters.
      */
+    template<typename FUNCTION_TYPE>
```

Comment:
Same template constraint and naming of template parameter as above.

Reply:
The constraints were already implemented and I changed the name in [8009ba3](https://github.com/ad-freiburg/qlever/pull/860/commits/8009ba3301f0cff3fa8441b2dd94f7785ce57fb6).